.vib/phpmyadmin/vib-platform-verify.json

```diff
@@ -19,7 +19,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/phpmyadmin"
         },
-        "runtime_parameters": "ZGI6CiAgaG9zdDogbWFyaWFkYgogIGFsbG93QXJiaXRyYXJ5U2VydmVyOiB0cnVlCiAgcG9ydDogMzMwNgogIGJ1bmRsZVRlc3REQjogdHJ1ZQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cHM6IDQ0MwptYXJpYWRiOgogIGF1dGg6CiAgICByb290UGFzc3dvcmQ6IHJvb3RQYXNzd29yZAo=",
+        "runtime_parameters": "ZGI6CiAgaG9zdDogbWFyaWFkYgogIGFsbG93QXJiaXRyYXJ5U2VydmVyOiB0cnVlCiAgcG9ydDogMzMwNgogIGJ1bmRsZVRlc3REQjogdHJ1ZQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODAKbWFyaWFkYjoKICBhdXRoOgogICAgcm9vdFBhc3N3b3JkOiByb290UGFzc3dvcmQ=",
         "target_platform": {
```

Comment:
Changing the port to HTTP on purpose. I've noticed some issues in the HTTPS setup I had before. There was a problem when loading phpmyadmin, an error pointing to a bad request. My conclusion is that phpMyAdmin doesn't support this 'fake' HTTP setup. 
Since fixing this was not the goal of the task but an accidental finding, I reverted to HTTP.